### DISCUSSION TITLE
Is it possible to serve AdminUI using a base url?

### DISCUSSION THREAD

**Discussion by vonba** [keystonejs/keystone#5804] [Questions] [answered]
I'm in a scenario where Keystone needs to live in a sub directory, e.g. /backend

Currently I'm getting 404 errors for the Next JS chunks and also the Keystone API because everything is expected to live in the root.

It looked like Next itself has a config option to set baseUrl (I think that's what it's called), but I haven't been able to find this option in Keystone.

**Comment by bladey** [comment]
Hey @vonba, currently Keystone 6 cannot be placed in a sub-directory, however we will take this into consideration, thank you!

**Reply by sergkoudi** [reply]
@bladey Any updates on this topic? I also need to configure this.

**Comment by dcousens** [comment]
Related https://github.com/keystonejs/keystone/pull/7135

**Comment by pc-erin** [comment, marked as answer]
This seems to have been added by #8545
